lib/support/cli.js

```diff
@@ -718,6 +723,12 @@ export function parseCommandLine() {
       describe:
         'Use the portable visual-metrics processing script (no ImageMagick dependencies).'
     })
+    .option('visualMetricsKeyColor', {
+      type: 'array',
+      nargs: 8,
```

Comment:
Think this is missmatch to the gist you provide with how to run it? I tried with `bin/browsertime.js --visualMetrics true --video true --iterations 1 --visualMetricsKeyColor poster 120 250 250 20 0.75 --visualMetricsKeyColor first_frame 250 20 20 20 0.75 --visualMetricsKeyColor second_frame 20 20 250 20 0.75 --visualMetricsKeyColor last_frame 255 255 50 50 0.75 --browser firefox --video.framerate 60 https://www.sitespeed.io` (I think that is in the example) but then I get "Not enough arguments following: visualMetricsKeyColor". Can you share how I can try it + do you know a live site where I can try?

Reply:
I updated my gist at https://gist.github.com/aosmond/4daa941cf36670e2ab0f7ecbeb66ed00. I don't have a live site. It's primary purpose is to measure the latency of video playback, hence the special rainbow video I made, but in theory any video that displays a non-white solid color frame at some point would work.